docs/Queries.md

```diff
@@ -90,8 +121,14 @@ A query response is also a plain object
   ref: string,
   value: any,
   type?: string,
-  flags?: array,
-  meta?: object,
+  meta?: {
+    flags?: string[],
+    variants?: {
+      [string]: {  // experiment
+        [string]: string  // context: variant
```

Comment:
nitpick, but context generally seems to be an integer right now? But, maybe that won't always be true?